src/main/resources/data/minecolonies/recipes/largequarry.json

```diff
@@ -20,6 +20,6 @@
     }
   },
   "result": {
-    "item": "minecolonies:largequarry"
+    "item": "minecraft:largequarry"
```

Comment:
What?

Reply:
typo

Reply:
Well, you've commented out the actual block, so this file probably just needs to be deleted since its target item isn't valid.

Reply:
made it .disabled so we remember to change that when adding the large